test/governance/voting.users.test.ts

```diff
@@ -34,56 +36,58 @@ beforeAll(async () => {
   sudo = UserFactory.createUser(Users.SudoUser, keyring, bootnode) as SudoUser;
 });
 
+beforeEach(async () => {
+  // Initialize actors
+  candidate = UserFactory.createUser(
+    Users.GovernanceUser,
+    keyring,
+    bootnode
+  ) as GovernanceUser;
+
+  voter = UserFactory.createUser(
+    Users.GovernanceUser,
+    keyring,
+    bootnode
+  ) as GovernanceUser;
+
+  bank = new Bank(sudo);
+
+  // Fund accounts
+  await voter.addMGATokens(bank.sudoUser, new BN(Math.pow(10, 17).toString()));
+  await candidate.addMGATokens(
+    bank.sudoUser,
+    new BN(Math.pow(10, 16).toString())
+  );
+
+  // Subscribe to events
+  candidate.node.subscribeToUserBalanceChanges(candidate);
+  voter.node.subscribeToUserBalanceChanges(voter);
+});
+
 describe("Governance -> Voting -> Users", () => {
-  test("Users can vote for a new council", async () => {
-    const candidate = UserFactory.createUser(
-      Users.GovernanceUser,
-      keyring,
-      bootnode
-    ) as GovernanceUser;
-
-    const voter = UserFactory.createUser(
-      Users.GovernanceUser,
-      keyring,
-      bootnode
-    ) as GovernanceUser;
-
-    const bank = new Bank(sudo);
-
-    await voter.addMGATokens(
-      bank.sudoUser,
-      new BN(Math.pow(10, 17).toString())
-    );
-    await candidate.addMGATokens(
-      bank.sudoUser,
-      new BN(Math.pow(10, 16).toString())
-    );
-    candidate.node.subscribeToUserBalanceChanges(candidate);
-    voter.node.subscribeToUserBalanceChanges(voter);
+  test("Users can vote for a new council member", async () => {
+    const stake = new BN(Math.pow(10, 14).toString());
 
+    // Gonzalo runs for council
+    // Eddy votes for Gonzalo
     await candidate.runForCouncil();
-    await voter.vote([candidate], new BN(Math.pow(10, 16).toString()));
+    await voter.vote([candidate], stake);
```

Comment:
can you validate at least one time that the voting is correctly done with the result of this request?
voting(AccountId): (BalanceOf,Vec<AccountId>)

Reply:
We know this happens because the outcome occurs and you see the amount in the voters balance

Reply:
Then we are missing coverage on this function. `voting(AccountId): (BalanceOf,Vec)`
or do you think that would be better to have a specific test for that guy?